subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -106,10 +107,12 @@ static void serial_cb(const struct device *uart, void *user_data)
 			break;
 		}
 
-		k_work_submit(&uart_tr->cb_work);
+		k_work_submit_to_queue(&uart_tr->rx_workq, &uart_tr->cb_work);
 	}
 }
 
+K_THREAD_STACK_DEFINE(uart_tr_workq_stack_area, 4096);
```

Comment:
Worth adding TODO: for moving it to uart_tr.

Reply:
done